Piwik.Tracker/PiwikTracker.cs

```diff
@@ -126,11 +126,11 @@ public class PiwikTracker
         /// <summary>
         /// Ecommerce item page view tracking stores item's metadata in these Custom Variables slots.
         /// </summary>
-        private const int CvarIndexEcommerceItemPrice = 2;
+        internal const int CvarIndexEcommerceItemPrice = 2;

```

Comment:
These ids are send in a json array to piwik server. Using a mocked server would also require to decompose the request url and search for these ids. (See also [Tracking HTTP API](https://developer.piwik.org/api-reference/tracking-api)->"Example Tracking Request"->"_cvar")